identity/app/views/profile/privacyForm.scala.html

```diff
@@ -104,15 +104,21 @@ <h2>Thank you</h2>
 @profile.emailPrefs(IdentityPage("/email-prefs", "Email preferences"), emailPrefsForm, emailSubscriptions, availableLists, idRequest, idUrlBuilder)
 
 @* CHANNELS *@
-@if(IdentityShowCommunicationChannelConsents.isSwitchedOn && communicationChannelsForUser(user).nonEmpty) {
+@if(IdentityShowCommunicationChannelConsents.isSwitchedOn) {
     <hr class="manage-account-divider" />
 
     <fieldset class="fieldset fieldset--manage-account-noborder">
         <div class="fieldset__heading">
-            <h2 class="form__heading">How can we get in touch with you?</h2>
+            <h2 class="form__heading">How else can we get in touch with you?</h2>
```

Comment:
is `else` needed?

Reply:
I added it because email is always on so to say, i feared not having `else` would imply there's no ways for guardian to get in touch with you